canonical-web-and-design/ubuntu.com
# Refactor /wsl tutorial section to pull tutorials from json endpoint

## Done

- Create a JSON endpoint that can be queried with a topic to only hold those tutorials
- Use JS to pull the data and display them in tutorial cards using html templates

## QA

- Go to /wsl and check all the tutorials are rendered, including one 'Docs' link as the last article.
- Check the p-divider__block classes are on the right tutorials
- Try with a different query to check it works dynamically

## Issue / Card

Fixes https://github.com/canonical-web-and-design/ubuntu.com/issues/11389
